src/command_modules/azure-cli-consumption/setup.py

```diff
@@ -33,7 +33,7 @@
 ]
 
 DEPENDENCIES = [
-    'azure-mgmt-consumption==1.1.0',
+    'azure-mgmt-consumption==2.0.0',
```

Comment:
Is it your intention to make this module GA?  If so, you will need to schedule a dedicated command review with the CLI team.